py3status/modules/deadbeef.py

```diff
@@ -3,95 +3,112 @@
 Display song currently playing in deadbeef.
 
 Configuration parameters:
-    cache_timeout: how often we refresh usage in seconds (default 1)
-    delimiter: delimiter character for parsing (default '¥')
-    format: see placeholders below (default '{artist} - {title}')
+    cache_timeout: refresh interval for this module (default 1)
+    delimiter: delimiter between metadata fields (default 'ÐÉÅÐƁÊËF')
+    format: display format for this module (default '[{artist} - ][{title}]')
 
 Format placeholders:
-    {artist} artist
-    {title} title
-    {elapsed} elapsed time
-    {length} total length
-    {year} year
-    {tracknum} track number
+    {artist}
+    {length}
+    {playback_time}
+    {title}
+    {tracknumber}
+    {year}
 
 Color options:
-    color_bad: An error occurred
+    color_pause: Paused, defaults to color_degraded
+    color_play: Playing, defaults to color_good
+    color_stop: Not Playing, defaults to color_bad
```

Comment:
Amended. Maybe we should avoid that kind of English discussion between verb, verb past tense, and verb present participle, and others... to keep options straightforward... especially for i8ln purpose.

Will change `state_pause` to `state_paused`.